test/tests/functional/pbs_multi_sched.py

```diff
@@ -1128,29 +1062,31 @@ def submit_jobs(self, num_jobs=1, attrs=None, user=TEST_USER):
         return ret_jids
 
     @skipOnCpuSet
-    def test_equiv_partition(self):
+    def test_equiv_queues(self):
         """
         Test the basic behavior of job equivalence classes: submit two
-        different types of jobs into 2 partitions and see they are
-        in four different equivalence classes
+        different types of jobs into 2 queues of same partition and see they
+        are in two different equivalence classes
         """
```

Comment:
I don't think this test is required.  There is no difference between equiv classes in one multi-sched and in the one default scheduler case.